code/modules/mob/mob_movement.dm

```diff
@@ -215,6 +215,10 @@
 /mob/Process_Spacemove(movement_dir = 0)
 	if(HAS_TRAIT(src, TRAIT_SPACEWALK) || spacewalk || ..())
 		return TRUE
+	var/turf/T = get_turf(src)
+	var/datum/gas_mixture/environment = T.return_air()
+	if(HAS_TRAIT(src, TRAIT_FLUTTER)&& (environment.return_pressure() > 30))
+		return TRUE
```

Comment:
You can edit this modularly by rewriting the proc in a `modular_splurt/code/modules/mob/mob_movement.dm` file, adding something like this:

```
/mob/Process_Spacemove(movement_dir = 0)
	var/turf/T = get_turf(src)
	var/datum/gas_mixture/environment = T.return_air()
	if(HAS_TRAIT(src, TRAIT_FLUTTER) && (environment.return_pressure() > 30))
		return TRUE
	. = ..()
```

Reply:
Does anything need to be done to the original, Or do I just need to make the file and put my code there?

Reply:
I suppose delete this edit and put the code in the file

Reply:
Added the new file and defined it in the DME. So this should be good now.